apps/train_loc/lib/train_loc/vehicles/vehicle.ex

```diff
@@ -41,7 +41,7 @@ defmodule TrainLoc.Vehicles.Vehicle do
           vehicle_id: non_neg_integer,
           timestamp: DateTime.t(),
           block: String.t(),
```

Comment:
question: since both this and `trip` go through `process_trip_block/1` should this also include `:unassigned`?